utils/db-parts.stanza

```diff
@@ -4,11 +4,16 @@ defpackage ocdb/db-parts :
   import collections
   import json
   import math
+  import lang-utils
+
   import jitx with :
     prefix(Resistor) => EModel-
     prefix(Capacitor) => EModel-
     prefix(Inductor) => EModel-
   import jitx/commands
+  import jitx/param-sets
```

Comment:
Do not import ESIR compiler internals. Wrap the interface you need in `jitx/commands`.